client/reader/test/utils.js

```diff
@@ -8,30 +8,37 @@ import { showSelectedPost } from '../utils';
 jest.mock( 'page', () => ( {
 	show: jest.fn(),
 } ) );
```

Comment:
This mock needs to be adapted to how we now call `page( url )` instead of `page.show( url )`

Reply:
Of course 👍  Done in https://teamcity.a8c.com/viewLog.html?buildId=7961706&buildTypeId=calypso_RunAllUnitTests